scanner/metadata_ffmpeg.go

```diff
@@ -74,14 +75,17 @@ func LoadAllAudioFiles(dirPath string) (map[string]os.FileInfo, error) {
 }
 
 func ExtractAllMetadata(inputs []string) (map[string]*Metadata, error) {
+	mds := map[string]*Metadata{}
 	cmdLine, args := createProbeCommand(inputs)
 
 	log.Trace("Executing command", "arg0", cmdLine, "args", args)
 	cmd := exec.Command(cmdLine, args...)
-	output, _ := cmd.CombinedOutput()
-	mds := map[string]*Metadata{}
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return mds, errors.New(fmt.Sprintf("error extracting metadata files: %s", err))
```

Comment:
You can use `fmt.Errorf()` instead of `errors.New(fmt.Sprintf())` to create a formatted error message